web-client/src/components/Avatar.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import picture from '../images/blank-profile-picture.png';
+import Button from "@material-ui/core/Button";
+import { withStyles } from "@material-ui/core/styles";
+//import Resizer from 'react-image-file-resizer';
+
+const useStyles = theme => ({
+    main:
+    {
+        width: '100%',
+        marginBottom: '5%',
+    },
+    btn:
+    {
+        marginLeft: '8%',
+        marginTop: '6%',
+        float: 'left',
+        textTransform: 'none',
+    },
+    btn1:
+    {
+        marginTop: '8%',
+        float: 'left',
+        textTransform: 'none',
+    },
+    photo:
+    {
+       width: 100,
+       height: 100,
+       margin: 'auto',
+       borderRadius: '50px',
+       marginLeft: '8%',
+    },
+    img:
+    {
+        width: '35%',
+        float: 'left',
+    }
+});
+
+
+class Avatar extends Component {
+  state={
+    profileImg: picture
+  }
+
+  imageHandler = (e) => {
+    const reader = new FileReader();
+    var file = e.target.files[0];
+
+    // const resizeFile = (file) => new Promise(resolve => {
```

Comment:
If this fragment of code is unnecessary, delete it pls

Reply:
which fragment? 51 Line?

Reply:
51-56
+ 5
+ 59
lines

Reply:
This is for resizing the avatar. This code will be corrected later so it's needed.

Reply:
But it's still unused code that does nothing but take up space. So you should delete these lines (later we can easily get it from git)

Reply:
Next time when someone has changes that doesn't work properly or there aren't complete yet and can be added to another PR. It can be stored in git stash.

Reply:
Moreover, package.json doesn't contain `react-image-file-resizer`, so there will be an error after uncommenting :C